test/gtest/ucp/test_ucp_perf.cc

```diff
@@ -16,6 +16,9 @@ class test_ucp_perf : public ucp_test, public test_perf {
 public:
     using ucp_test::get_ctx_params;
 protected:
+    virtual void init() {
+        test_base::init(); /* Skip entities creation in ucp_test */
+    }
```

Comment:
@yosefe 
"i don't think we want to force creating 2 entities for every ucp test"
I completely agree with you. We can avoid this useless creation by the method provided by this design. I implemented `entity` creation not in the class ctor() but in the initialization procedure provided by gtest. So, if we don't need pre-allocated two entities we can implement particular test class ::init() function and **doesn't** call intermediate init() from `ucp_test`.
Some tests needs only one entity to work. These test uses sender() function only.